src/skmultiflow/trees/nodes/lc_learning_node_nba.py

```diff
@@ -0,0 +1,65 @@
+from skmultiflow.bayes import do_naive_bayes_prediction
+from skmultiflow.trees.nodes import LCLearningNodeNB
+
+
+class LCLearningNodeNBA(LCLearningNodeNB):
```

Comment:
Please add a short description.

Reply:
Description added in commit [5f331dc](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/5f331dce48293665f7ef78b059ba2d6782606438).